lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Is this leftover from the lint changes?

Reply:
Oops. Yeah for some reason it was yelling at me for not having return types for API calls, so I added that in there while I worked. Will remove it. Thanks!